changelogs/current.yaml

```diff
@@ -57,6 +57,9 @@ minor_behavior_changes:
     get route config factories by the configuration proto full names by default. This behavior change
     can be reverted by setting the ``envoy.reloadable_features.get_route_config_factory_by_type``
     runtime flag to false.
+- area: dubbo
+  change: |
+    add metadata_match in :ref:`RouteConfiguration <envoy_v3_api_msg_extensions.filters.network.dubbo_proxy.v3.RouteAction>` support to the dubbo proxy.
```

Comment:
I think this is new feature rather than behaviour change? 🤔

Reply:
get it